FeedKit/images/ImageRepository.swift

```diff
@@ -399,15 +410,16 @@ extension ImageRepository: Images {
       )
 
       os_log("loading: %@", log: log, type: .debug, url.lastPathComponent)
-
-      Nuke.loadImage(with: req, options: opts, into: imageView) {
-        response, error in
-        dispatchPrecondition(condition: .onQueue(.main))
-
-        if let er = error {
-          os_log("image loading failed: %{public}@", log: log, er as CVarArg)
+      
+      imageView.nk.setImage(with: req, options: opts) { result in
+        switch result {
+        case .failure(let er):
+           os_log("image loading failed: %{public}@", log: log, er as CVarArg)
```

Comment:
I'm planning to integrate `os_log` in Nuke 8.1. I started with `os_signpost` in Nuke 8.0 as a replacement for a custom solution for measuring performance – `ImageTaskMetrics.

Reply:
👍 Good, making sure to load the correct image (size) while making no unnecessary requests, I observe these logs quite a bit